src/main/java/au/csiro/fhir/claml/FhirClamlService.java

```diff
@@ -174,12 +176,30 @@ protected CodeSystem claml2FhirObject(ClaML claml, List<String> displayRubrics,
             cs.setValueSet(valueSet);
         }
 
-        if (claml.getIdentifier().size() > 1) {
-            log.warn("Multiple identifiers not currently supported by FHIR Code Systems");
-        }
         for  (Identifier ident : claml.getIdentifier()) {
             cs.addIdentifier(new org.hl7.fhir.r4.model.Identifier().setSystem(ident.getAuthority()).setValue(ident.getUid()));
         }
+        if (Strings.isNotBlank(identifiers)) {
```

Comment:
You might consider using Option.Builder.hasArgs() to support multiple values instead of parsing yourself.